routes/api/users.js

```diff
@@ -37,4 +38,10 @@ router.post(
   userControllers.logout
 );
 
+// The Routes for the user Updating the account
+router.put('/users/profile/:id/update', passport.authenticate('jwt', { session: false }), multerConfig, userControllers.updateProfile);
+
+// The Route to get the user profile
```

Comment:
Authentication on this route should be optional, as everyone should be able to view the profile without being able to update the content. Consider the [readme](https://github.com/andela/ah-kgl-avengers-backend#get-profile) file

Reply:
Thank you @avpaul I come to realize that the authentication here is not necessary and I have removed it.